### PR TITLE
Aghost Now Can Drop Things Wherever They Want

### LINKED ISSUE #19225: Admin observer can only drop items at default range.
## Description
They can pickup items from anywhere visible but can only drop items at the default maximum range.

**Reproduction**
1. `aghost`.
2. pickup item.
3. try to drop it far away.

### PR DESCRIPTION
**Pull request by Geekyhobo**
## About the PR
Aghosts can now drop things wherever they please

## Why / Balance
Closes #19225, Admin QOL

## Technical details
Adds a tag that can be added that ignores the range and obstruction check when something leaves a players hands

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
- 
https://github.com/space-wizards/space-station-14/assets/66805063/f09ee58d-f219-4c68-9453-a1a62f741879

## Breaking changes

**Changelog**

:cl: 
- fix: Fixed AGhosts not being able to drop items past the default range.

### CODE REVIEW

**Comment by GreyMario** [comment]
label: Holy Shit

(this has made me very angry across many hours of testing things in-game localhost)

File: Resources/Changelog/Admin.yml [hunk 1]
```diff
@@ -81,3 +81,8 @@ Entries:
   - {message: 'Fixed not being able to right click in AHelps and the players and objects tabs.', type: Fix}
   id: 13
   time: '2023-12-21T06:34:00.0000000+00:00'
+- author: Geekyhobo
+  changes:
+    - {message: 'Fixed AGhosts not being able to drop items past the default range.', type: Fix}
+  id: 13
```
Comment (metalgearsloth) [inline Resources/Changelog/Admin.yml:87]:
```suggestion
  id: 14
```

**Comment by Geekyhobo** [comment]
Thanks for the fix

File: Resources/Prototypes/tags.yml [hunk 2]
```diff
@@ -223,6 +223,9 @@
 - type: Tag
   id: BypassInteractionRangeChecks
 
+- type: Tag
+  id: BypassDropChecks
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:228]:
```suggestion
- type: Tag
  id: BypassDropChecks

- type: Tag
  id: BypassInteractionRangeChecks

```

File: Resources/Prototypes/tags.yml [hunk 3]
```diff
@@ -221,8 +221,10 @@
   id: BulletFoam
 
 - type: Tag
-  id: BypassInteractionRangeChecks
+  id: BypassDropChecks
 
+- type: Tag
+  id: BypassInteractionRangeChecks
 - type: Tag
```
Comment (metalgearsloth) [inline Resources/Prototypes/tags.yml:228]:
```suggestion

- type: Tag
  id: BypassInteractionRangeChecks
  
- type: Tag
```

I'm gonna slap github

**Pull request merged**